doc/src/modules/functions/elementary.rst

```diff
@@ -1,78 +1,130 @@
 Elementary
 ==========
 
-This module implements elementary functions, as well as functions like ``Abs``,
-``Max``, etc.
+This module implements elementary functions such as trigonometric, hyperbolic
+as well as functions like ``Abs``, ``Max``, ``sqrt`` etc.
```

Comment:
According to Wikipedia, the set of elementary functions is all single-variable functions that can be built from `exp`, `log`, roots, constants, and arithmetic operators. This includes trigonometic functions and their inverses via the complex identities.
If that's the correct definition, then `sqrt` need not be explicitly mentioned because it's already elementary.

Also, `Abs`, `Max` etc. are not elementary. According to Wikipedia, they're "basic special functions". If that's correct, they really belong either into `special` or into `special.basic`. I suppose that that's going beyond the intended scope of this PR; the Great Function Reclassification project is going to be a large-ish effort because it will involve dealing with import cycles.